app/controllers/va_facilities_controller.rb

```diff
@@ -1,5 +1,7 @@
 class VaFacilitiesController < ApplicationController
   include PracticeUtils
+  helper_method :created_practices_count
+  helper_method :adopted_practices_count
```

Comment:
Do we need these? `helper_method` is used for calling a method from a helper module in a controller.

Reply:
Yeah - these methods are in the controller and being called from a view (facilities_index table)... bc I can't access the session helper - is_user_a_guest? method in the model - where we were previously getting the counts.